stage1/enter_kvm/enter_kvm.go

```diff
@@ -38,6 +40,92 @@ var (
 	appName string
 )
 
+// fileAccessible checks if the given path exists and is a regular file
+func fileAccessible(path string) bool {
+	if info, err := os.Stat(path); err == nil {
+		return info.Mode().IsRegular()
+	}
+	return false
+}
+
+func sshPrivateKeyPath() string {
+	return filepath.Join(kvmSettingsDir, kvmPrivateKeyFilename)
+}
+
+func sshPublicKeyPath() string {
+	return sshPrivateKeyPath() + ".pub"
+}
+
+// generateKeyPair calls ssh-keygen with private key location for key generation purpose
+func generateKeyPair(private string) error {
```

Comment:
Good idea to do it later (and also use ssh lib to make connection, what would remove ssh as client dependency).